ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Validators/CreateProjectDTOValidator.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace ExadelTimeTrackingSystem.Data.Validators
+{
+    using ExadelTimeTrackingSystem.BusinessLogic.DTOs;
+    using FluentValidation;
+
+    public class CreateProjectDTOValidator : AbstractValidator<CreateProjectDTO>
+    {
+        public CreateProjectDTOValidator()
+        {
+            RuleFor(p => p.Name)
+                .NotEmpty();
+
+            RuleFor(p => p.ApproverId)
+                .NotEmpty();
+
+            RuleFor(p => p.Activities)
+                .NotEmpty();
+
+            RuleForEach(p => p.Activities)
+                .NotEmpty();
+
+            RuleForEach(p => p.EmployeeIds)
+                .NotEmpty();
+
+            RuleFor(p => p.Name)
+                .MaximumLength(50);
```

Comment:
you don't need to specify separate RuleFor for Name again. You already have it on line 10, just add .MaximumLength after .NotEmpty